test-suite/schema-generation/collapsible/collapsible-no-op_metaschema.xml

```diff
@@ -11,7 +11,7 @@
    <define-assembly name="parent">
       <formal-name>parent</formal-name>
       <description>parent assembly</description>
-      <root-name>parent</root-name>
+      <root-name>collapsible-no-op-parent</root-name>
```

Comment:
I didn't see a change on accompanying data files, so I assume this lives up to its name?